python/py-telethon/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-telethon
+version             1.24.0
+revision            0
+license             MIT
+maintainers         nomaintainer
+
+description         Full-featured Telegram client library.
+long_description    Telethon is an asyncio Python 3 MTProto library to \
+                    interact with Telegram's API as a user or through \
+                    a bot account (bot API alternative).
+
+homepage            https://github.com/LonamiWebs/Telethon
+
+python.versions     37 38 39 310
+
+distname            [string totitle ${distname}]
```

Comment:
instead, please set `python.rootname Telethon` - without the Tcl magic please, just write it out for us mortals to understand ;) That will also make sure that the `livecheck.name` is automatically set correctly.

Reply:
Thanks done.